examples/widgets_minimal/src/qpushbutton_cxx.rs

```diff
@@ -0,0 +1,41 @@
+// SPDX-FileCopyrightText: 2023 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+#[cxx_qt::bridge(cxx_file_stem = "qpushbutton_cxx")]
+mod ffi {
+
+    unsafe extern "C++Qt" {
+        include!(<QtWidgets/QPushButton>);
+        type QPushButton;
+
+        include!("cxx-qt-lib/qstring.h");
+        type QString = cxx_qt_lib::QString;
+
+        #[qsignal]
+        #[allow(dead_code)]
```

Comment:
as we only connect to the signal and don't emit it causes dead code, wonder if we should always mark as dead ?

Reply:
(as this is an existing signal rather than one being created from Rust, so for inherit / C++Qt types you might not see an emit)

Reply:
Isn't required anymore